Gems/Multiplayer/Code/Source/Debug/MultiplayerDebugMultiplayerMetrics.cpp

```diff
@@ -0,0 +1,222 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Source/Debug/MultiplayerDebugMultiplayerMetrics.h>
+#include <Multiplayer/IMultiplayer.h>
+
+namespace Multiplayer
+{
+#ifdef IMGUI_ENABLED
+    void AccumulatePerSecondValues(const MultiplayerStats& stats, const MultiplayerStats::Metric& metric, float& outCallsPerSecond, float& outBytesPerSecond)
```

Comment:
nit: Did you consider passing / returning struct to future proof this if we wanted to accumulate other stats?